lib/jazzy/podspec_documenter.rb

```diff
@@ -0,0 +1,109 @@
+require 'tmpdir'
+require 'json'
+
+module Jazzy
+  class PodspecDocumenter
+    attr_reader :podspec
+
+    def initialize(podspec)
+      @podspec = podspec
+    end
+
+    def sourcekitten_output
+      sandbox = Pod::Sandbox.new(pod_config.sandbox_root)
+      installer = Pod::Installer.new(sandbox, podfile)
+      installer.install!
+      stdout = Dir.chdir(sandbox.root) do
+        pod_targets.map do |t|
+          SourceKitten.run_sourcekitten(
+            %W(doc --module-name #{podspec.module_name} -target #{t}),
```

Comment:
should wrap `#{podspec.module_name}` in single quotes in case it contains a space